app/epa/settings.py

```diff
@@ -162,10 +162,13 @@
 
 CRISPY_TEMPLATE_PACK = "bootstrap4"
 
-if DEBUG is True:
-    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-else:
+USE_SMTP_EMAIL_BACKEND = ast.literal_eval(

```

Comment:
```suggestion
USE_SMTP_EMAIL_BACKEND = not ast.literal_eval(

```

Reply:
Good catch eagle eye :) This was a typo in the environment variable name (had a focus on using the console backend before, changed it to smtp and forgot to rename this one instance).

I fixed this in a commit bb7dabf, in which I simply renamed `USE_CONSOLE_EMAIL_BACKEND` to `USE_SMTP_EMAIL_BACKEND`.

I like this solution more than adding a `not` operator in front of the literal evaluation result. Code seems IMO clearer this way.

I also added a commit d9de92a, in which I set SMTP mailing as default so that you don't have to change anything in production. This means if you want to use console backend locally (or in production) you have to `export USE_SMTP_EMAIL_BACKEND=False`

Reply:
I agree with your decisions :)